lib/boku/utils.py

```diff
@@ -42,3 +42,19 @@ def error(msg):
         error('did not verify: {0} != {1}'.format(real, amount))
 
     log.info('transaction: {0} verified successfully'.format(transaction))
+
+
+def check_sig(data):
+    """
+    Check the signature in the request.
+    """
+    raw = data.copy()
+    boku_sig = raw.pop('sig')
+
+    client = get_client(settings.BOKU_MERCHANT_ID, settings.BOKU_SECRET_KEY)
+    our_sig = client.check_sig(get_pairs_string(raw))['sig']
+
+    if our_sig != boku_sig:
+        msg = 'signature check failed'
```

Comment:
is there an identifier we can log here too? Something like 

    'signature check failed for {t}'.format(t=data.get('transaction_id', '<unknown transaction>'))